definitions/variable/CO2.yaml

```
@@ -0,0 +1,5 @@
+# This yaml file adds needed variables for E3M´s model data upload
+
+- Capacity|CO2|Direct Air Capture:
+    description: Installed (available) capacity to extract CO2 using Direct Air Capture
```

Suggestion to revise the description
```suggestion
    description: Installed (available) capacity to extract CO2 from the atmosphere
      using Direct Air Capture (DAC)
```